### PR TITLE
translations: removal of i18n support

### LINKED ISSUE #12: deploy: fix compile_catalog issue
```
running compile_catalog
error: [Errno 2] No such file or directory: 'invenio_celery/translations/'
ValueError: Cannot find file (or expand pattern): 'dist/*'
running upload_docs
```

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* Removes i18n support from the module as it is not used.
  (closes #12)

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] include `messages.pot`
- [ ] include `*.po` and `*.pot` in `MANIFEST.in`

**Comment by lnielsen** [comment]
Just one observation - we don't have any translation strings in the module. Should we 1) remove all translation related code from the module (i.e. in case we later add translation strings) or 2) just don't provide the ``invenio_i18n.translations`` entry point to ensure the empty translation is not loaded.

**Comment by jirikuncar** [comment]
For this package it seems that we can go without I18N support.

- [ ] remove `.tx` folder
- [ ] remove `invenio_i18n.translations` entry point
- [ ] amend `.travis.yml:deploy` section
- [ ] check dependencies, `_` functions, and `invenio_celery/translations` folder

**Comment by SamiHiltunen** [comment]
i18n support should be completely removed now

File: docs/Makefile [hunk 1]
```diff
@@ -16,10 +16,8 @@ endif
 PAPEROPT_a4     = -D latex_paper_size=a4
 PAPEROPT_letter = -D latex_paper_size=letter
 ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
-# the i18n builder cannot share the environment and doctrees with the others
-I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
```
Comment (jirikuncar) [inline docs/Makefile]: @SamiHiltunen this part is for building internationalized documentation. Please keep it untouched.

File: docs/make.bat [hunk 2]
```diff
@@ -7,10 +7,8 @@ if "%SPHINXBUILD%" == "" (
 )
 set BUILDDIR=_build
 set ALLSPHINXOPTS=-d %BUILDDIR%/doctrees %SPHINXOPTS% .
-set I18NSPHINXOPTS=%SPHINXOPTS% .
```
Comment (jirikuncar) [inline docs/make.bat]: https://github.com/inveniosoftware/invenio-celery/pull/14/files#r44384296

File: docs/Makefile [hunk 3]
```diff
@@ -19,7 +19,7 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 
-.PHONY: help clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest coverage gettext
```
Comment (jirikuncar) [inline docs/Makefile]: keep this file untouched

File: docs/make.bat [hunk 4]
```diff
@@ -9,8 +9,8 @@ set BUILDDIR=_build
 set ALLSPHINXOPTS=-d %BUILDDIR%/doctrees %SPHINXOPTS% .
 set I18NSPHINXOPTS=%SPHINXOPTS% .
 if NOT "%PAPER%" == "" (
-	set ALLSPHINXOPTS=-D latex_paper_size=%PAPER% %ALLSPHINXOPTS%
```
Comment (jirikuncar) [inline docs/make.bat]: keep this file untouched

**Pull request merged**